src/draw_bot/handlers/action.cr

```diff
@@ -48,4 +48,23 @@ module DrawBot
     role_name_guard,
     CannedResponse.new("data/spray",
       template: "$author sprays $content with a $spray"))
+
+  client.on_message_create(
+    SplitParser.new("~pokeball", join_after: 1, min_args: 1)
+  ) do |payload, context|
+    args = context[DrawBot::SplitParser::Results].arguments
+    user = args[0]
```

Comment:
Strictly speaking, the input text can be anything. In these style of commands, I'd like to call the argument `subject` instead of `user` - since it's just a `String` that we have no expectations of.

If we actually did something like parsed the string into a `User`, that would be a different story.